packages/claims/test/claims.test.ts

```diff
@@ -26,11 +26,14 @@ describe('[CLAIMS PACKAGE/CLAIMS]', function () {
   before(async () => {
     const registry = await deployRegistry([userAddress]);
     const store = new DidStore(await spawnIpfsDaemon());
-    const owner = withKey(withProvider(signerFromKeys(keys), getProvider()), walletPubKey);
-    const userDoc = new DIDDocumentFull(
-      userDdid,
-      new Operator(owner, { address: registry }),
+    const owner = withKey(signerFromKeys(keys).connect(getProvider()), walletPubKey);
+    const operator = new Operator(
+      keys.privateKey,
+      { address: registry },
+      keys.publicKey,
+      'http://localhost:8544',
     );
+    const userDoc = new DIDDocumentFull(userDdid, operator);
     claims = new Claims(owner, userDoc, store);
```

Comment:
This should be changed as, currently, the use of the `withKey` function still requires an `ethers` Signer, which we don't want.
I think what we should do is the delete `IdentityOwner` interface in `did-resolver-interface` and instead rely everywhere on the `IdentityOwner` from `did-ethr-resolver`.